mvoRender/examples/vbg-gridwall-lazyload-onscroll.yml

```diff
@@ -1,572 +0,0 @@
-_version: "3"
-_name: VZ Gridwall
-_metadata:
-  base_url: https://www.verizon.com
-  controlsEditable: false
-  treeShakeDependencies: false
-  enableSass: false
-  dependencies: []
-  deviceSettings:
-    device:
-      show-by-default: true
-      title: Laptop with HiDPI screen
-      screen:
-        horizontal:
-          width: 1440
-          height: 900
-        device-pixel-ratio: 2
-        vertical:
-          width: 900
-          height: 1440
-      capabilities: []
-      user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.97 Safari/537.36
-      type: notebook
-      modes:
-        - title: default
-          orientation: horizontal
-          insets:
-            left: 0
-            top: 0
-            right: 0
-            bottom: 0
-    orientation: horizontal
-    zoomFactor: 1
-web:
-  _metadata: {}
-  _config:
-    dependencies: ""
-  l1w5sso1m:
-    _id: l1w5sso1m
-    _display_name: Gridwall
-    _metadata:
-      page_def:
-        domain_match_type: full
-        path_match_type: simple
-        reference_urls:
-          - https://www.verizon.com/business/shop/products/devices/smartphones
-        _pattern: business/shop/products/devices/smartphones
-      script: "
-
-        /*****
-
-        \ * Context code
-
-        ****/
-
-        var rule = window.evolv.renderRule.expBP
-
-        var $ = rule.$;
-
-        var store = rule.store;
-
-
-        store.instrumentDOM({
-
-        \    gridwallWrapper: {
```

Comment:
You might want to resave this YML so that it doesn't escape each tab.

Reply:
that file was deleted